README.md

```diff
@@ -10,10 +10,11 @@ Sessions are:
   * [Session 01: Web Basics & Browser Security Model](01-web-basics-part-1/README.md)
   * [Session 02: Cookies & Session Management & Access Control](02-web-basics-part-2/README.md)
   * [Session 03: Broken Access Control](03-broken-access-control/README.md)
+  * [Session 03: Securing Communication](08-secure-communication/README.md)
```

Comment:
Should this line be like this?

I see that it's session 03 but it points to session 08-secure-communication.